crates/control/src/ball_filter.rs

```diff
@@ -97,6 +104,11 @@ impl BallFilter {
         context: &CycleContext,
     ) -> Vec<Hypothesis> {
         for (detection_time, balls) in measurements {
+            let cycle_time = detection_time
```

Comment:
```suggestion
            let last_cycle_duration = detection_time
```
? What do you think?